src/app/find-support/find-support.component.html

```diff
@@ -15,10 +15,9 @@
 -->
 
 <div class="header">
-  <h1 class="title">Support Resources</h1>
+  <h1 class="title">Resource Kit</h1>
   <p class="subtitle">
-    Processing harmful content can be tough, but you're not alone in this
-    process. There are people and tools to support you each step of the way.
+    Find skills, tools and partner .
```

Comment:
_Find skills, tools and partner ._ or `Find skills, tools and partners.` Please confirm before merging.